src/components/Agreed/Agreed.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { spacingSmall, spacingMedium } from '../../styles/designTokens';
+import media from '../../styles/media';
+import Paragraph from '../Layout/Paragraph';
+
+const Wrapper = styled.section`
+  margin: ${spacingSmall} auto;
+
+  ${media.tablet`
+    margin: ${spacingMedium} auto;
+  `};
+`;
```

Comment:
better would be to wrap application by [`theme-provider`](https://www.styled-components.com/docs/advanced) and provide global styles.
so here you would define `${props => props.theme.spacing.small}` instead of referencing `../../styles` this is not necessary and gives you better flexibility